### PR TITLE
add teal mask details

### PR DESCRIPTION
**Pull request by WyattC16**
Adding new details from the teal mask dlc

### CODE REVIEW

File: data/v2/csv/pokemon.csv [hunk 1]
```diff
@@ -1280,3 +1287,7 @@ id,identifier,species_id,height,weight,base_experience,order,is_default
 10269,miraidon-drive-mode,1008,28,2400,,,0
 10270,miraidon-aquatic-mode,1008,28,2400,,,0
 10271,miraidon-glide-mode,1008,28,2400,,,0
+10272,bloodmoon-ursaluna,901,24,2900,,,0
```
Comment (giginet) [inline data/v2/csv/pokemon.csv]: Pokémon identifier should be started with species name.

So it should be `ursaluna-bloodmoon`

**Review by giginet — COMMENTED** [review]
Some abilities of past-generation Pokémon were also changed. (e.g. Empoleon, Shiftry)

File: data/v2/csv/pokemon.csv (hunk 1 above)
Comment (WyattC16) [reply data/v2/csv/pokemon.csv]: I've corrected this. Nice catch

Comment (WyattC16) [reply data/v2/csv/pokemon.csv]: I also updated the abilities of Empoleon and Shiftry. I didn't see any others that changed.

**Comment by pokeapi-machine-user** [comment]
A [PokeAPI/api-data](https://github.com/PokeAPI/api-data) refresh has started. In 45 minutes the staging branch of [PokeAPI/api-data](https://github.com/PokeAPI/api-data/tree/staging) will be pushed with the new generated data. <br><br> The staging branch will be deployed in our [staging environment](https://staging.pokeapi.co/api/v2/) and you will be able to review the entire API. <br><br> A Pull Request ([master](https://github.com/PokeAPI/api-data/tree/master)<-[staging](https://github.com/PokeAPI/api-data/tree/staging)) will be also created at [PokeAPI/api-data](https://github.com/PokeAPI/api-data/pulls) and assigned to the PokeAPI Core team to be reviewed. If approved and merged new data will soon be available worldwide at [pokeapi.co](https://pokeapi.co/api/v2/).

**Comment by pokeapi-machine-user** [comment]
The updater script has finished its job and has now opened a Pull Request towards [PokeAPI/api-data](https://github.com/PokeAPI/api-data/pulls) with the updated data. <br><br> You can see the Pull Request deployed at our [staging environment](https://staging.pokeapi.co/api/v2/) when [CircleCI deploy](https://app.circleci.com/pipelines/github/PokeAPI/deploy) will be finished (_check the started time of the last build_).

**Comment by giginet** [comment]
Thank you to resolve my review.

But, the change is not applied to the pre-evolved Pokémon of Empoleon. (Piplup, Prinplup)

I'll fix this later. Thank you!

**Pull request merged**